src/main/java/org/testng/reporters/XMLConstants.java

```diff
@@ -42,6 +44,9 @@
   /** failures attribute for testsuite elements */
   String ATTR_FAILURES = "failures";
 
+  /** ignored attribute for testsuite elements */
+  String ATTR_IGNORED = "ignored";
```

Comment:
I don't think that introducing a new xml element called `ignored` is a good idea. Because it is not compliant with a junit report produced by Ant/ Maven Surefire. Both of them for disabled tests uses the `<skipped />` element.